src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -0,0 +1,75 @@
+package com.kodilla.ecommercee.domain;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "ORDERS")
+public class Order {
+
+    @Id
+    @NotNull
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long order_id;
+
+    @NotNull
+    @Column(name = "order_price")
+    private Double order_price;
+
+    @NotNull
+    @Column(name = "products_quantity")
+    private Integer order_quantity;
```

Comment:
Domyślam się, że w linii 25 też planowałeś mieć "products_quantity", ale w sumie inne nazwy to nie błąd, więc chyba może już zostać

Reply:
Dzięki za feedback! W encji Order według mnie bardziej pasuję ilość produktów bo encja Order chyba nie powinna przetrzymywać ilości zamówień tylko klasa np. User. A zmienną nazwałem order_quantity żeby pasowała do reszty. Mogę zmienić jak to koliduję z jakimiś "good patterns" programistycznymi.

Reply:
Jezeli kolumne nazywasz products_quantity to podobnie nazwij  encji.